docs/tutorials/getting-started-with-csharp/WeatherMicroservice/README.md

```diff
@@ -0,0 +1,27 @@
+C# Microservices Sample
+================
+
+This sample is created during the [Creating Microservices hosted in Docker](../microservices.html)
+tutorial. Please see that topic for detailed steps on the code
+for this sample.
+
+Key Features
+------------
+
+This sample demonstrates creating a web service using asp.net core and C#
+and deploying that service in a Docker Container.
+
+Build and Run
+-------------
```

Comment:
Each readme would benefit from a single link to first install the .NET Core SDK.